samples/04_gis_analysts_data_scientists/counting_features_in_satellite_images_using_scikit_image.ipynb

```diff
@@ -26,57 +26,49 @@
    "outputs": [],
```

Comment:
<p>Can agol variable name be changed to gis, since python api playground is enterprise and not AGOL?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>